app/frontend/src/features/connections/friends/AddFriendButton.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { Empty } from "google-protobuf/google/protobuf/empty_pb";
+import React, { useState } from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { service } from "../../../service";
+import wrapper from "../../../test/hookWrapper";
+import AddFriendButton from "./AddFriendButton";
+
+const sendFriendRequestMock = service.api.sendFriendRequest as jest.Mock<
+  ReturnType<typeof service.api.sendFriendRequest>
+>;
+
+function TestComponent() {
```

Comment:
Since the add friend mutation is inlined, there's no easy way other than to test the whole button component as UI tests. But I think that's fine too by the [second reason stated here](https://react-hooks-testing-library.com/#when-not-to-use-this-library)